build.gradle

```diff
@@ -89,6 +89,11 @@ project('common') {
 def withoutLogger = { exclude group: 'org.slf4j', module: 'slf4j-log4j12'
                       exclude group: 'org.slf4j', module: 'slf4j-simple' }
 
+project('authplugin') {
```

Comment:
`authplugin` needs to be added to `publishAllJars` so that it can be imported as its own separate artifact by implementors right?

Reply:
Fixed.